src/app/pages/ElectronicDelivery.jsx

```diff
@@ -41,8 +41,10 @@ class ElectronicDelivery extends React.Component {
       selectedItem && selectedItem.itemSource ? selectedItem.itemSource : null;
     const raiseError = _isEmpty(this.props.error) ? {} : this.props.error;
     const serverRedirect = true;
+    const isEddRequestable = selectedItem.eddRequestable
 
     this.state = _extend({
+      isEddRequestable,
```

Comment:
Vedy nieece!

Reply:
oh thats actually a vestige of the redux stuff lol